test/openAPI/features/wallet_linked_authenticate.feature

```diff
@@ -0,0 +1,47 @@
+@method=POST @endpoint=/linked-authorization/authenticate
+Feature: The endpoint to check the correctness of the data.
+
+  @smoke @unit @positive
+  Scenario: Successfully checks the correctness of the data smoke type test
+    Given Wants to check the correctness of the data
+    And The link code is generated before POST /linked-authorization/authenticate
+    When Send POST /linked-authorization/authenticate request with given requestTime, linkedTransactionId, individualId, "PIN" as authFactorType, "password" as challenge, "alpha-numeric" as format
```

Comment:
what auth factor types are supported is based on a implementation of BB, instead of hardcording to PIN, can we keep this configurable ?

Reply:
This is a good idea but is hard to implement it because we do not have any other software in the repo ... For now we can keep it hardcoded and later change it

Reply:
Hardcoding is ok for now, but we should have a feature to input a configuration file to test harness based on BB implementation which changes the test slightly for optional elements